packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -187,18 +189,21 @@ class CheckboxListTile extends StatelessWidget {
         break;
     }
     return new MergeSemantics(
-      child: ListTileTheme.merge(
-        selectedColor: activeColor ?? Theme.of(context).accentColor,
-        child: new ListTile(
-          leading: leading,
-          title: title,
-          subtitle: subtitle,
-          trailing: trailing,
-          isThreeLine: isThreeLine,
-          dense: dense,
-          enabled: onChanged != null,
-          onTap: onChanged != null ? () { onChanged(!value); } : null,
-          selected: selected,
+      child: new Theme(
+        data: themeData,
```

Comment:
this is probably overly expensive given how big ThemeData is... maybe Checkbox should expose an outerConstraints so we can set it directly? (Same with the other list tile widgets.)

Reply:
instead of supporting an arbitrary constraint, I could make all of these widgets take a MaterialTapTargetSize in the constructor.  If it isn't provided, then we can look it up from the Theme.

Reply:
Done